neotools/cli.py

```diff
@@ -10,7 +10,7 @@
 logger = logging.getLogger(__name__)
 
 file_index_arg = partial(click.argument, 'file_index',
-                         type=click.IntRange(1, 8))
+                         type=click.IntRange(1))
```

Comment:
Why drop the upper bound?

Reply:
At least my device, the neo2, can have an unlimited number of files, not just the 8. They are identified by the index number (see the screenshot I posted here: https://github.com/lykahb/neotools/issues/1#issuecomment-809006172 where the first file (named "f" for some reason) does not have a "space" - it is not bound to any key.

Reply:
That's right, it can have many saved files. But the ones assigned to the keys have an index and can be addressed.

Reply:
So any number higher than eight wouldn't work.

Reply:
Interesting - on the neo2, all the files can be addressed - after my changes, I was able to read all the files into a local directory:
```
🤔 ls -l
total 44
-rw-r--r-- 1 egh egh   43 Mar 28 19:06 agerev.txt
-rw-r--r-- 1 egh egh  381 Mar 28 19:06 artoutline.txt
-rw-r--r-- 1 egh egh  541 Mar 28 19:06 art.txt
-rw-r--r-- 1 egh egh   78 Mar 28 19:06 f.txt
-rw-r--r-- 1 egh egh  528 Mar 28 19:06 hamilton.txt
-rw-r--r-- 1 egh egh  739 Mar 28 19:06 noshortcuts.txt
-rw-r--r-- 1 egh egh 4692 Mar 28 19:06 novel.txt
-rw-r--r-- 1 egh egh   21 Mar 28 19:06 priceofpeace.txt
-rw-r--r-- 1 egh egh 4714 Mar 28 19:06 trillion.txt
```

Reply:
```
/home/egh/.local/share/virtualenvs/neotools/bin/neotools files read 9
Characters
...
```

Reply:
I mean that the files can be read but their index does not show up anywhere in the UI of Neo, so the indexes feel arbitrary when used. At the moment neotools can only read the unbound files with command `files read all`.

You are right that neotools has a bug conflating the space and file index. But I doubt that exposing the file index to CLI is right because it is a very confusing concept. If not all keys are bound, I think that file can be for example, bound to key 5 and have the index 1. What do you think of a new invocation format where the read command takes either the space or the name?

Reply:
Yes, I think that makes more sense. It will, I think, require retrieving all file attributes from the Neo before fetching a file, in order to look up by file name or space. But that seems worth it to me - it would be a lot clearer. I'll update the PR for that, if you like.

Reply:
This looks like the right approach. There is a possibility of a file named with a single digit. This would cause a collision between the space and name. The chance for that is small and it probably does not makes sense to complicate the CLI commands for that case. A single digit can just default to the space. An updated PR would be great, thanks!

Reply:
I've updated the PR to allow referencing files in read/write/clear by the name or space.